package.json

```diff
@@ -3,6 +3,10 @@
 	"version": "2.2.0",
 	"description": "JS language shims used by Airbnb.",
 	"main": "index.js",
+	"files": [
```

Comment:
The “files” field is dangerous - the only way published files should ever be restricted is by npmignore.